src/components/settings/Users/routes.js

```diff
@@ -0,0 +1,29 @@
+import { h } from "vue";
```

Comment:
The intent is to store routes used by a specific feature where the feature itself stands. This is a good practice for maintainability, compare to have all routes in a single global file (imagine what happens in a bigger webapp 🤯). In addition, this follow the spirit of vuejs, which keep template (html), js and CSS of a component all at the same place (SFC: Single File Component). So why the related routes should be far away the corresponding feature? IMHO that make no sense...

Also, none of the other settings use this pattern because... none of the other settings have children (from router point of view). I guess it's a good enough reason for that difference. wdyt?

Reply:
Given that all other routes are stored centrally, I think it would be "easier to merge", if this follows the established patterns
=> this be merged into the central router.

I understand that this makes sense and think the router should be refactored, but lets keep this PR atomic ^^

Reply:
Sad, but okay

Reply:
One could also argue that having a separate routes file would reduce the chance of conflicts, making it easier to merge. This is also very clear to read as is.

Reply:
@CommanderStorm I just have a question before doing the requested change: since that file is only about the feature i'm working on, why do you consider it as "not atomic"? Since I'm not refactoring anything and it seems this pattern please you and @chakflying too, why not keeping it as a first step before doing the router refactoring you speak about?

TBH, I'm a bit sad to change something which:
- don't impact anything else (= "atomic")
- looks like the thing to do anywhere else in the futur 😢

Reply:
I would think that such a change should be done over the whole codebase
=>  at the same time (=in one PR)
=> keeping the codebase in a self-consistent state

Being atomic i the context of PRs is not really about impact of other code/coders, it is more about keeping the cognitive load on the side of the Reviewer managable
=> keeping [PRs small+focused](https://stevegrunwell.github.io/code-review-for-me-and-you/#/5/3)
=> sticking to established patterns helps in this case, as otherwise the Reviewer (in this case Louis) would need to understand if this really is equivalent and if this is a thing that should be added.
=> I would think this change would fit better into another PR, which is more focused on said problem.

That being said, you are free to overrule this, I don't think this will be a big merge-blocker for Louis

Reply:
> I would think that such a change should be done over the whole codebase

I agree!

> => at the same time (=in one PR)

I agree too, but only for existing code

> => keeping the codebase in a self-consistent state

Yes and no... I think it's a trad-off because in practice it's not always possible/interesting. Here I think it's not relevant because of the small impact on the code base + if it please to Louis, someone will have to spend time to achieved the code I already have... really bad for velocity isn't it? 🙃 

However, I understand your argues and even found them really pragmatic 👍🏼 
Since this change is easy to do, and, as you stand yourself, probably not a merge-blocker, my proposal is to keep it as is for now. If Louis request a change, I will just have to move the code in the global router file, should take me something like 5 minutes... well, not a big deal... 😉